src/sales/transactions.md

```diff
@@ -2,11 +2,81 @@
 title: Transactions
 ---
 
-The _Transactions_ grid lists all payment activity that has taken place between your store and a payment system, and provides access to more detailed information.
+The _Transactions_ grid lists all payment activity that has taken place between your store and a payment system. Each record provides access to more detailed information.
 
 ## View transactions
 
 On the _Admin_ sidebar, go to **Sales** > _Operations_ > **Transactions**.
 
 ![]({% link images/images/sales-transactions.png %}){: .zoom}
 _Transactions grid_
+
+### Column descriptions
+
+|Column|Description|
+|--- |--- |
+|ID|A unique numeric identifier that is assigned for each transactional record.|
```

Comment:
Please alphabetize things when possible: This table and the list of keys, even if it differs from the order in the admin.

Reply:
Thank you @dobooth !
Done, please approve :)